website/docs/getting_started/conceptual_overview.md

```diff
@@ -0,0 +1,112 @@
+- A specific version of the consumer will always have exactly one pact version for each provider.
+
+## Branches & Environments
+
+:::info THE GOLDEN RULE FOR LABELLING
+Label with the branch name when you publish pacts or verification results, and record deployment with the environment name after you deploy.
+
+- [branches](/pact_broker/branches)
+- [recording deployments and releases](/pact_broker/recording_deployments_and_releases)
+
+::: 
+
+At any point you may want to label a particular version of the consumer with one or more identifiers. For example, 
+you can identify the consumer with its current branch (e.g. a PR branch or the main branch), or you can identify it with the 
+environment it has been deployed to.
+
+Branches in the Pact Broker are designed to model repository (git, svn etc) branches. A branch in the Pact Broker belongs to a pacticipant (application). A branch may have many pacticipant versions, and a pacticipant version may belong to many branches (but typically, it will belong to just one).
+
+Remember that a pacticipant version in the Pact Broker should map 1:1 to a commit in your repository. To facilitate this, the version number used to publish pacts and verification results should either be or contain the commit.
+
+We recommend that you set the branch property when you publish pacts and verification results, and use record-deployment or record-release when you deploy/release.
+
+The general recommendation is to set the branch name when you publish a pact, and set with the environment when you deploy.
+I'll discuss later how we use labels to help answer the question: can I deploy?
+
+For example, let's say you have a PetStoreConsumer application.  Every time you change it that's a new version 
+of the application.  When you run contract tests for that version, you publish a new pact publication to the broker, 
+which may or may not generate a new pact version.
+
+| Date  | Consumer Versions | branches     | environment | Pact Version |
+| ----- | ----------------- | ------------ | ----------- | ------------ |
+| 12/21 | 63be8b            | prod, main   | production  | 1            |
+| 12/22 | 9ecb61            | staging,main | staging     | 2            |
+| 12/23 | 31696e            | feat-xyz     | N/A         | 2            |
+| 12/23 | 3b4de7            | feat-abc     | N/A         | 3            |
+
+Here we see that:
+- Version 63be8b is deployed to prod and is in the main branch.
+- Version 9ecb61 is deployed to staging, and is also in the main branch. It has changed the pact, so it has a new pact version.
+- Version 31696e is in a feature branch and has not been deployed anywhere. It did not change the pact, so the pact version remains the same
+- Version 3b4de7 is in a different feature branch. It did change the pact, so we now have a new pact version
+  
+
+:::warning A caveat about tags
```

Comment:
we now haven't mentioned tags above, so not sure if this should be brought up.

or should it something like, familiar with Pact and using tags, see the below notice/warning.